spec/models/hmpps_api/offender_spec.rb

```diff
@@ -325,9 +325,9 @@
         end
 
         context 'without a new LDU mapping' do
```

Comment:
We should probably remove references to a "new LDU" mapping.

There should be no such thing as "old" and "new" LDUs after this PR.